src/puppetlabs/trapperkeeper/services/webrouting/webrouting_service_core.clj

```diff
@@ -24,6 +24,9 @@
 (def ContextHandlerOptions
   (dissoc (merge jetty9-core/ContextHandlerOptions RouteOption) :server-id))
 
+(def WebsocketHandlerOptions
+  (dissoc (merge jetty9-core/CommonOptions RouteOption) :server-id))
```

Comment:
On reading this again I agree that `RouteOption` would fit fine, I was just copying the pattern that seemed apparent from `add-servlet-hanlder`, but as we have no specific options then sure, just passing RouteOption would make sense.